### PR TITLE
Attack point (multiple damage pr attack fix)

### PR DESCRIPTION
**Pull request by turell4k**

### CODE REVIEW

File: my_game.py [hunk 1]
```diff
@@ -47,7 +47,7 @@
 PLAYER_SIGHT_RANGE = SCREEN_WIDTH/4 # How far can the player see?
 
 # Amount of players
-NUM_OF_PLAYERS = 2
+NUM_OF_PLAYERS = 1
```
Comment (tobiasbp) [inline my_game.py:50]:
```suggestion
NUM_OF_PLAYERS = 2
```

File: my_sprites.py [hunk 2]
```diff
@@ -319,6 +319,9 @@ def __init__(self,
         # angle used for the dir the sprite is facing. for the enemy this will usually be the angle to the target
         self._direction = 0  # direction facing. Should be in degrees
 
+        # point which actually hits target during attack
+        self.attack_point = None
```
Comment (tobiasbp) [inline my_sprites.py:323]: This would need to go into the _Weapon_. And probably be a list (think of a hand grenade)

File: my_game.py (hunk 1 above)
Comment (turell4k) [reply my_game.py:50]: This is neccesary for debugging, since enemies don't work with multiplayer. Will be reverted tho

File: my_game.py [hunk 3]
```diff
@@ -308,6 +307,12 @@ def on_update(self, delta_time: float = 1/60):
             # Updates the player_sprite_list.
             p.update()
 
+        for e in self.tilemap.sprite_lists["enemies"]:
```
Comment (tobiasbp) [inline my_game.py:310]: Add comment here pls

File: my_sprites.py [hunk 4]
```diff
@@ -244,6 +247,12 @@ def rate(self):
     def attacks_left(self):
         return self._attacks_left
 
+    def attack_successful(self, victim):
+        # Can't assert 'victim' as an Entity, because it has not been defined yet
+        # Subtract health points from target and remove attack point
+        victim.hp -= self.strength
```
Comment (tobiasbp) [inline my_sprites.py:253]: Let the main loop responsible for removing the HP from the entity that was hit.

**Review by tobiasbp — CHANGES_REQUESTED** [review]
Minor changes neeed.

**Review by tobiasbp — APPROVED** [review]

**Pull request merged**